src/com/leetcode/problems/HighestAltitude.java

```diff
@@ -0,0 +1,18 @@
+package com.leetcode.problems;
+
+import java.util.Arrays;
+
+public class HighestAltitude {
+  public static void main(String[] args) {
+    System.out.println(largestAltitude(new int[] {-4, -3, -2, -1, 4, 3, 2}));
+  }
+
+  public static int largestAltitude(int[] gain) {
+    int currentPoint = 0, highestAlt = 0;
```

Comment:
`highestAlt` **do not use short nouns without a good reason, full nouns are always preffered.**